File: cakk-domain/src/main/java/com/cakk/domain/repository/jpa/CakeShopLinkJpaRepository.java
package com.cakk.domain.repository.jpa;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.cakk.domain.entity.shop.CakeShopLink;

public interface CakeShopLinkJpaRepository extends JpaRepository<CakeShopLink, Long> {

	List<CakeShopLink> findAllByCakeShopId(Long cakeShopId);
}

File: cakk-infrastructure/persistence-mysql/src/main/java/com/cakk/infrastructure/persistence/repository/jpa/CakeShopLinkJpaRepository.kt
package com.cakk.infrastructure.persistence.repository.jpa

import com.cakk.infrastructure.persistence.entity.shop.CakeShopLinkEntity
import org.springframework.data.jpa.repository.JpaRepository

interface CakeShopLinkJpaRepository : JpaRepository<CakeShopLinkEntity, Long> {

	fun findAllByCakeShopId(cakeShopId: Long): List<CakeShopLinkEntity>
}

